pulp_container/tests/functional/api/test_repositories_list.py

```diff
@@ -186,6 +194,7 @@ def test_none_user(self):
         repositories = self.get_listed_repositories(auth)
```

Comment:
I am a bit confused, should not user_none with toke_auth disabled see all 3 repos? (2 private and 1 public)

Reply:
Not because of this: https://github.com/pulp/pulp_container/blob/6d4b4f4912ec8542e90d139c83be551b74215c65/pulp_container/app/registry_api.py#L518

Reply:
aha, but then don't we state actually in our docs that the concept of private and public repo does not work if you disabled token auth?

Reply:
yes, but is not the interpretation of it like: admin can do everything, others can do nothing apart from pulling?

Reply:
seems like we have a discrepancy 
```
$ podman logout puffy.example.com
Error: not logged into puffy.example.com
$ http https://puffy.example.com/v2/_catalog  --ignore-netrc
HTTP/1.1 200 OK
Access-Control-Expose-Headers: Correlation-ID
Allow: GET, HEAD, OPTIONS
Connection: keep-alive
Content-Length: 49
Content-Type: application/json
Correlation-ID: 08fb832d01cd4df5945e3ad88ba740c8
Cross-Origin-Opener-Policy: same-origin
Date: Wed, 29 May 2024 15:20:44 GMT
Docker-Distribution-Api-Version: registry/2.0
Referrer-Policy: same-origin
Server: nginx
Strict-Transport-Security: max-age=15768000
Vary: Accept
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Registry-Supports-Signatures: 1

{
    "repositories": [
        "docker-cache/library/busybox"
    ]
}


$ podman pull puffy.example.com/rhosp-rhel8/openstack-cron:16.2
Trying to pull puffy.example.com/rhosp-rhel8/openstack-cron:16.2...
Getting image source signatures
Copying blob 2b814fe20317 skipped: already exists  
Copying blob b243f36441d6 skipped: already exists  
Copying blob 6f474dfa2585 skipped: already exists  
Copying config 6cb8a7d9e4 done  
Writing manifest to image destination
Storing signatures
6cb8a7d9e4333a95f5f6cd8a34d40b10ada4d4362096573f6e5e751b8ed9baba
$ pulp container distribution update --name 'rhosp-rhel8/openstack-cron' --public
Started background task /pulp/api/v3/tasks/018fc4f1-00ff-7f0a-8bce-0c7b5ab20409/
Done.
$ http https://puffy.example.com/v2/_catalog  --ignore-netrc
HTTP/1.1 200 OK
Access-Control-Expose-Headers: Correlation-ID
Allow: GET, HEAD, OPTIONS
Connection: keep-alive
Content-Length: 78
Content-Type: application/json
Correlation-ID: 3c6d43c6fa7c4f8d82c46be927682e30
Cross-Origin-Opener-Policy: same-origin
Date: Wed, 29 May 2024 15:21:24 GMT
Docker-Distribution-Api-Version: registry/2.0
Referrer-Policy: same-origin
Server: nginx
Strict-Transport-Security: max-age=15768000
Vary: Accept
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Registry-Supports-Signatures: 1

{
    "repositories": [
        "docker-cache/library/busybox",
        "rhosp-rhel8/openstack-cron"
    ]
}




```